Source/Core/VideoCommon/VertexLoaderARM64.cpp

```diff
@@ -136,23 +111,11 @@ int VertexLoaderARM64::ReadVertex(VertexComponentFormat attribute, ComponentForm
   int load_bytes = elem_size * count_in;
   int load_size = GetLoadSize(load_bytes);
   load_size <<= 3;
-  elem_size <<= 3;
 
-  if (offset == -1)
-  {
-    if (count_in == 1)
-      m_float_emit.LDR(elem_size, IndexType::Unsigned, coords, EncodeRegTo64(scratch1_reg), 0);
-    else
-      m_float_emit.LD1(elem_size, 1, coords, EncodeRegTo64(scratch1_reg));
```

Comment:
I'm a bit confused as to how this code previously worked (probably because I don't quite understand what `LD1` does). Where is `count_in` being used in the `LD1` case?